### DISCUSSION TITLE
New Hardware

### DISCUSSION THREAD

**Discussion by SRLive** [TheLazyGeekGuy/DIY-Waves-Soundgrid-Server#6] [Show and tell]
Just so you know, I have been using for quite some times now a Lenovo thinkcenter M920Q with a i5-8500T and it's just perfect.

**Comment by matieckhardt** [comment]
Hi!  SounGrid Server workin on an oldie AMD Phenom Six core.

![Screenshot 2023-04-16 at 21 12 27](https://user-images.githubusercontent.com/41755571/232351812-369739f4-256b-4c11-97bf-bf96adca3dff.png)
![Screenshot 2023-04-16 at 21 12 36](https://user-images.githubusercontent.com/41755571/232351815-9b1e01f7-0460-4f49-af80-8fc113cbcbeb.png)
![Screenshot 2023-04-16 at 21 12 52](https://user-images.githubusercontent.com/41755571/232351817-6593b249-5005-4f92-ac27-cd4b8ec3c26a.png)
![Screenshot 2023-04-16 at 21 13 09](https://user-images.githubusercontent.com/41755571/232351818-614e3dd6-1503-4d3b-881b-be2605267029.png)

next test with Amd  ryzen 7 4700s. (ps5 cheap mobo [combo)
